### DISCUSSION TITLE
TimeInput with strange format

### DISCUSSION THREAD

**Discussion by rodolphoasb** [mantinedev/mantine#658] [Q&A] [answered]
Hey, first I want to thank you all for this awesome library... I even pretend to contribute to it in the future!

I'm trying to use TimeInput inside a component but I'm getting this strange format:

![image](https://user-images.githubusercontent.com/8127858/149379182-a2546275-a9ce-4df0-9add-4b35ee692c23.png)

Does anyone have any idea why this might be happening?

**Comment by rtivital** [comment, marked as answer]
You might be providing an invalid date. Here is a working example – https://codesandbox.io/s/admiring-black-t52jc?file=/src/App.tsx
